addons/web_editor/models/field_html_history_mixin.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+from .patch_utils import apply_patch, generate_comparison, generate_patch
+
+
+class HtmlHistory(models.AbstractModel):
+    _name = "field.html.history.mixin"
+    _description = "Field html History"
+
+    history_revision_ids = fields.One2many(
+        "field.html.history.revision", "res_id", string="Related revision Ids",
+        copy=True)
```

Comment:
Don't think you want to copy revision history by default. Generally when copying, especially for models like knowledge, it is more to start with a template / validated content than copying the history that is behind it.

Reply:
True. good catch.